root/admin/wikidoc/CreateWikiDoc.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *      \  á))  s  .'
+ *       '-.((*_.-'
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormRowText from '../../components/FormRowText';
+import FormSubmit from '../../components/FormSubmit';
+import Layout from '../../layout';
+import {withCatalystContext} from '../../context';
+
+type Props = {
+  +$c: CatalystContextT,
+  +form: FormT<{
+    +page: FieldT<string>,
+    +version: FieldT<string>,
+  }>,
+};
+
+const CreateWikiDoc = withCatalystContext(({
+  $c,
+  form,
+}: Props) => (
+  <Layout fullWidth title={l('Add Page')}>
+    <div id="content">
+      <h1>{l('Add Page')}</h1>
+      <form action={$c.req.uri} method="post">
+        <FormRowText
```

Comment:
The original template used `form_row_text_long`, and it does look quite small for most page titles now.
```suggestion
        <FormRowTextLong
```

Reply:
Restored in https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/bb22262479c52846e0c550fce2b25b2a2bd39e3f